README.md

```diff
@@ -300,7 +345,14 @@ compatible.both(Future.of('world')).value(console.log);
 ### Creating Futures
 
 #### Future
-##### `Future :: ((a -> (), b -> ()) -> Cancel) -> Future a b`
```

Comment:
Yeah, those lines were never intended to be used as links, and the documentation never linked to them. I'm okay with breaking the deep links out there that people may have generated, I don't expect there to be many.